applications: serial_lte_modem: introduce POWER_PIN (#14112)
nrfconnect/sdk-nrf: applications/serial_lte_modem/doc/slm_description.rst

   This option makes an nRF91 Series device enter deep sleep after startup.
   It is not selected by default.

.. _CONFIG_SLM_POWER_PIN:

CONFIG_SLM_POWER_PIN - Interface GPIO pin to power off the SiP and exit from sleep or idle

>    This option specifies which pin to use to power on/off the SiP as well as make SLM exit idle mode.

Reviewer: ```suggestion
   This option specifies which pin to use to power on or off the SiP and make SLM exit idle mode.
```
Previous line only says "power off". Which line is correct?

   * On Thingy:91, **P0.26** (Multi-function button on Thingy:91) is used.

   .. note::
      This pin is configured with a pull up and so is active low.

>       It must be pulled down shortly to perform one power off or wake up operation.

Author: Maybe not quite. The intended meaning is that it must be pulled down for a short time.
